draft-ietf-rats-corim.md

```diff
@@ -1591,129 +1590,117 @@ The Verifier SHOULD set the `authorized-by` field in Accepted Claims Set entries
-`environment-map` as the Reference Value. These are the candidate claims. If there are
-no candidate claims then the Reference Value does not match.
+#### Processing Multi-Environment Conditional (MEC) Endorsement Triple
 
-A Verifier SHALL compare two `environment-map`s using a binary comparison of the CBOR
-encoded objects.
+For each MEC Endorsement Triple the Verifier compares each of the `stateful-environment-record` fields from the `cond` field in the triple against the ACS (see {{sec-match-one-se}}).
 
-A Verifier SHOULD convert `environment-map` into a form which meets CBOR Core
-Deterministic Encoding Requirements {{-cbor}} before performing the binary comparison.
+If every stateful environment matches then the Verifier MUST add an Endorsement entry to the ACS (see {{sec-add-to-acs}}) for each `endorsed-triple-record` in the `endorsements` field.
+Each endorsement is created from the `endorsed-triple-record` and the authority which signed the CoRIM file containing the MEC Endorsement Triple.
 
-If the Reference Value contains an `authorized-by` field then the Verifier
-SHALL modify the candidate claims set to remove Claims whose `authorized-by`
-field does not contain one of the keys listed in the Reference Value
-`authorized-by` field (see {{sec-authorized-by}} for more details).
-If all candidate claim entries are discarded by this step then the
-Reference Value does not match.
+#### Processing Conditional Endorsement Series Triple
 
-The Verifier SHALL iterate over the codepoints which are present in the
-`measurement-values-map` field within the Reference Value `measurement-values-map`.
-The Reference Value entry is compared against each of the candidate claims.
-If none of the candidate claims matches
-the Reference Value entry then the Reference Value does not match.
+For each Conditional Endorsement Series Triple the Verifier iterates over the `conditional-series-record`s within the triple, stopping if it finds a match.
 
-The algorithm used to match the `measurement-values-map` entries
-is described below. The comparison performed depends on the type of
-field being compared.
+For each iteration, the Verifier creates a temporary `stateful-environment-record` by merging the `stateful-environment-record` in the triple with the `refv` field in the `conditional-series-record`. It compares this temporary record against the ACS (see {{sec-match-one-se}}).
 
-If the Reference Value `measurement-values-map` value is tagged with a CBOR
-tag {{-cbor}} then the Verifier MUST use the comparison algorithm associated
-with that tag.
+If one of the temporary records matches then the Verifier MUST add an Endorsement entry to the ACS.
+This Endorsement is created from the `environment-map` field in `stateful-environment-record`, the `endv` field from the `conditional-series-record` which matched and the authority which signed the CoRIM file containing the Conditional Endorsement Series Triple.
```

Comment:
Is this line needed? It seems to restate the matching algorithm described above. A simple change to line 1653 seems sufficient to describe the nature of the endorsement value added.